### PR TITLE
Fix dark mode detection and container height

### PR DESCRIPTION
**Pull request by pngwn**
## Description

- Fixes darl mode on spaces
- fixes the page height (fitters bit at the bottom on main)

Closes: #(issue)

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

**Comment by pngwn** [comment]
https://huggingface.co/spaces/pngwn/dakr

File: js/app/src/routes/[...catchall]/+page.svelte [hunk 1]
```diff
@@ -235,52 +235,52 @@
...
-		const dark_class_element = is_embed ? target.parentElement! : document.body;
-		const bg_element = is_embed ? target : target.parentElement!;
-		bg_element.style.background = "var(--body-background-fill)";
-		if (theme === "dark") {
-			dark_class_element.classList.add("dark");
-		} else {
-			dark_class_element.classList.remove("dark");
-		}
-	}
+	// function handle_theme_mode(target: HTMLDivElement): "light" | "dark" {
```
Comment (abidlabs) [inline js/app/src/routes/[...catchall]/+page.svelte:238]: Is this intentionally commented out?

Comment (pngwn) [reply js/app/src/routes/[...catchall]/+page.svelte:238]: Nope. I'll delete that.

**Review by abidlabs — APPROVED** [review]
looks good from my tests @pngwn!

**Pull request merged**